MarketDataMessage2.capnp

```diff
@@ -0,0 +1,108 @@
+using Cxx = import "/capnp/c++.capnp";
+using import "Exchanges.capnp".Exchange;
+
+$Cxx.namespace("proto::marketdata");
+@0xb58df63b06ce5a55;
+
+
+#######################################################################################################
+#                   COMMON TYPES
+#######################################################################################################
+
+
+struct Header {
+    clientID @0 :UInt64;
+    senderCompID @1 :Text = "<UNDEFINED>";
+}
+
+
+enum Chanel {
+	ticker @0;          # ticker data
+    orderbook @1;       # level 2 orderbook data
+}
+
+
+enum Side {
```

Comment:
Is side one of the data structures that is refactored into a "common" capnp file in the future? Since it's the same as the data structure in TradeMessage?

Reply:
for now we can import it from TradeMessage?

Reply:
This is the plan after we get stable version 1 of protocol. Common part will be excluded to a separate file.